### ISSUE TITLE
Playwright Tests are failing on 0.27.1

### ISSUE THREAD

**Issue by mofojed** [deephaven/web-client-ui#1466]
Creating a ticket to track the Playwright test failures and any information gathered.

An example failure on `main` is: https://github.com/deephaven/web-client-ui/actions/runs/6003447550

**Comment by mofojed** [comment]
It's possible to fairly intermittently reproduce the failing test case with rollup rows on a Mac running the e2e tests in Docker:
1. Restrict Docker resources to use only 2 cores, and 7GB of RAM, to match [GitHub hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources):
<img width="862" alt="image" src="https://github.com/deephaven/web-client-ui/assets/4505624/8ca35636-bfbf-4a04-b07c-1b77ea2422ec">
2. Run the e2e tests from the command line; specify the rollup rows test and turn on traces so you can see all the debug information : `DHC_VERSION=0.27.1 npm run e2e:docker -- --trace=on --grep "rollup rows and aggregrate columns" `
3. When the tests fail (they don't fail _every_ time), open up the report (playwrite-report/index.html), open the failing test, download the trace and then open it: `npx playwright show-trace <trace-filename>`
4. Look in the trace for errors. In this case, it looks like tables are disconnecting randomly for some reason:
<img width="1728" alt="image" src="https://github.com/deephaven/web-client-ui/assets/4505624/01011c66-5068-4b1a-8db9-76f054dd95d5">

The Docker logs for the server do not show anything noteworthy, just repeating:
```
2023-08-28 15:57:34 2023-08-28T19:57:34.693Z | rStateTracker.Driver |  INFO | i.d.e.t.i.u.RuntimeMemory | Jvm Heap: 495,704,672 Free / 830,472,192 Total (4,294,967,296 Max)
2023-08-28 15:57:40 2023-08-28T19:57:40.075Z | EFAULT.refreshThread |  INFO | e.u.i.PeriodicUpdateGraph | Minimal Update Graph Processor cycle times: 29.627ms / 12 cycles = 2.469ms/cycle average), safePointTime=0ms
2023-08-28 15:57:52 2023-08-28T19:57:52.089Z | EFAULT.refreshThread |  INFO | e.u.i.PeriodicUpdateGraph | Minimal Update Graph Processor cycle times: 25.040ms / 12 cycles = 2.087ms/cycle average), safePointTime=0ms
2023-08-28 15:58:09 2023-08-28T19:58:09.110Z | EFAULT.refreshThread |  INFO | e.u.i.PeriodicUpdateGraph | Minimal Update Graph Processor cycle times: 25.458ms / 17 cycles = 1.498ms/cycle average), safePointTime=0ms
2023-08-28 15:58:25 2023-08-28T19:58:25.140Z | EFAULT.refreshThread |  INFO | e.u.i.PeriodicUpdateGraph | Minimal Update Graph Processor cycle times: 27.201ms / 16 cycles = 1.700ms/cycle average), safePointTime=0ms
2023-08-28 15:58:27 2023-08-28T19:58:27.081Z | heduler-Concurrent-2 |  INFO | portedTableUpdateListener | ExportedTableUpdateListener(46edab78) is complete
2023-08-28 15:58:34 2023-08-28T19:58:34.699Z | rStateTracker.Driver |  INFO | i.d.e.t.i.u.RuntimeMemory | Jvm Heap: 492,803,152 Free / 830,472,192 Total (4,294,967,296 Max)
2023-08-28 15:58:36 2023-08-28T19:58:36.167Z | EFAULT.refreshThread |  INFO | e.u.i.PeriodicUpdateGraph | Minimal Update Graph Processor cycle times: 26.209ms / 11 cycles = 2.383ms/cycle average), safePointTime=0ms
```

**Comment by mofojed** [comment]
An example of a trace file failing (from a run where I tried to update Playwright to 1.37.1).
[c1c9588f4791c409f450e668351b61fc62b7c1c7.zip](https://github.com/deephaven/web-client-ui/files/12457970/c1c9588f4791c409f450e668351b61fc62b7c1c7.zip)

**Comment by mofojed** [comment]
There seems to a NPE thrown in the MultiplexWebSocketStream that isn't logged because it could get us stuck in a loop: https://github.com/deephaven/deephaven-core/blob/0724b2be3012278593375e945b670aaad44ed834/grpc-java/grpc-servlet-websocket-jakarta/src/main/java/io/grpc/servlet/web/websocket/MultiplexedWebsocketStreamImpl.java#L114

Stack trace of the NPE is:
```
2023-08-29 14:54:16 2023-08-29T18:54:16.013Z | qtp880533231-33      |  WARN | plexedWebsocketStreamImpl | MJB MultiplexedWebsocketStreamImpl cancel: Status{code=UNKNOWN, description=null, cause=java.lang.NullPointerException: Cannot invoke "io.grpc.internal.CompositeReadableBuffer.readableBytes()" because "this.unprocessed" is null
2023-08-29 14:54:16     at io.grpc.internal.MessageDeframer.readRequiredBytes(MessageDeframer.java:346)
2023-08-29 14:54:16     at io.grpc.internal.MessageDeframer.deliver(MessageDeframer.java:269)
2023-08-29 14:54:16     at io.grpc.internal.MessageDeframer.deframe(MessageDeframer.java:178)
2023-08-29 14:54:16     at io.grpc.internal.AbstractStream$TransportState.deframe(AbstractStream.java:210)
2023-08-29 14:54:16     at io.grpc.internal.AbstractServerStream$TransportState.inboundDataReceived(AbstractServerStream.java:262)
2023-08-29 14:54:16     at io.grpc.servlet.web.websocket.AbstractWebsocketStreamImpl$WebsocketTransportState.inboundDataReceived(AbstractWebsocketStreamImpl.java:25)
2023-08-29 14:54:16     at io.grpc.servlet.web.websocket.AbstractWebsocketStreamImpl.inboundDataReceived(AbstractWebsocketStreamImpl.java:99)
2023-08-29 14:54:16     at io.grpc.servlet.web.websocket.MultiplexedWebSocketServerStream.onMessage(MultiplexedWebSocketServerStream.java:219)
2023-08-29 14:54:16     at io.grpc.servlet.web.websocket.AbstractWebSocketServerStream.lambda$onOpen$0(AbstractWebSocketServerStream.java:115)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.jakarta.common.messages.AbstractDecodedMessageSink.invoke(AbstractDecodedMessageSink.java:61)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.jakarta.common.messages.DecodedBinaryMessageSink.onWholeMessage(DecodedBinaryMessageSink.java:60)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.messages.ByteBufferMessageSink.accept(ByteBufferMessageSink.java:64)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.jakarta.common.messages.AbstractDecodedMessageSink.accept(AbstractDecodedMessageSink.java:80)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.jakarta.common.JakartaWebSocketFrameHandler.acceptMessage(JakartaWebSocketFrameHandler.java:592)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.jakarta.common.JakartaWebSocketFrameHandler.onBinary(JakartaWebSocketFrameHandler.java:642)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.jakarta.common.JakartaWebSocketFrameHandler.onFrame(JakartaWebSocketFrameHandler.java:243)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.WebSocketCoreSession$IncomingAdaptor.lambda$onFrame$1(WebSocketCoreSession.java:671)
2023-08-29 14:54:16     at org.eclipse.jetty.server.handler.ContextHandler.handle(ContextHandler.java:1466)
2023-08-29 14:54:16     at org.eclipse.jetty.server.handler.ContextHandler.handle(ContextHandler.java:1485)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.server.internal.AbstractHandshaker$1.handle(AbstractHandshaker.java:212)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.WebSocketCoreSession$IncomingAdaptor.onFrame(WebSocketCoreSession.java:671)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.AbstractExtension.nextIncomingFrame(AbstractExtension.java:145)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.PerMessageDeflateExtension.nextIncomingFrame(PerMessageDeflateExtension.java:236)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.DemandingFlusher.emitFrame(DemandingFlusher.java:145)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.PerMessageDeflateExtension$IncomingFlusher.inflate(PerMessageDeflateExtension.java:480)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.PerMessageDeflateExtension$IncomingFlusher.handle(PerMessageDeflateExtension.java:413)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.DemandingFlusher.process(DemandingFlusher.java:169)
2023-08-29 14:54:16     at org.eclipse.jetty.util.IteratingCallback.processing(IteratingCallback.java:243)
2023-08-29 14:54:16     at org.eclipse.jetty.util.IteratingCallback.succeeded(IteratingCallback.java:369)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.DemandingFlusher.onFrame(DemandingFlusher.java:104)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.PerMessageDeflateExtension.onFrame(PerMessageDeflateExtension.java:93)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.ExtensionStack.onFrame(ExtensionStack.java:120)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.WebSocketCoreSession.onFrame(WebSocketCoreSession.java:481)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.WebSocketConnection.onFrame(WebSocketConnection.java:262)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.WebSocketConnection.fillAndParse(WebSocketConnection.java:455)
2023-08-29 14:54:16     at org.eclipse.jetty.websocket.core.internal.WebSocketConnection.onFillable(WebSocketConnection.java:340)
2023-08-29 14:54:16     at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314)
2023-08-29 14:54:16     at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100)
2023-08-29 14:54:16     at org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.runTask(AdaptiveExecutionStrategy.java:416)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.consumeTask(AdaptiveExecutionStrategy.java:385)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.tryProduce(AdaptiveExecutionStrategy.java:272)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy.lambda$new$0(AdaptiveExecutionStrategy.java:140)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:411)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:969)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1194)
2023-08-29 14:54:16     at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1149)
2023-08-29 14:54:16     at java.base/java.lang.Thread.run(Thread.java:833)
2023-08-29 14:54:16 }
```

Looks like the MessageDeframer is getting closed for some reason, and then another message is received on it. Trying to debug with Colin, but is so far only (mostly) consistently reproducible on my Mac when running the tests, and we can't trigger it locally on Ubuntu (even with restricting resources) so debugging is going slowly.

**Comment by mattrunyon** [comment]
Should be fully fixed with #1453
